content/installation/webserver/WebserverAgentConfig.md

```diff
@@ -0,0 +1,85 @@
+
+
+The items that must be configured are:
+* The communication link between Contrast-Service and TeamServer
+* The NGINX service enabling the Agent to inspect traffic to certain endpoints
+
+## Configure the Contrast Service
```

Comment:
@palmergs Is this different from the configuration instructions at https://docs.contrastsecurity.com/installation-service.html#service-config? If so, we should explain the difference and/or how they work together.

Reply:
🚧  It is almost the same. There's one additional configuration that is required for the Webserver agent.  `agent.service.socket = /run/contrast.sock` is required. Let me look at a separate PR to get that updated on that page.

Reply:
Differences that I see are the location of the contrast_security.yaml file and the fact that it configures a tcp port rather than a unix socket for the agent to speak to.
Personally, I would rather the all the webserver agent configuration be centrally located and described.  At this point, as a user it feels confusing to me if I'm trying to configure the webserver agent but I need to deal with settings potentially applicable to ruby and python agents.
@palmergs we also haven't even talked about the separate modsecurity configuration files.  Do I mention those at all?  I don't really want a user poking around them, but there is nothing to stop them.  What are you're thoughts about all this?

Reply:
discussed this with @palmergs.  We decided to treat the existing contrast-service doc as a "global" doc that discuss contrast-service configuration with respect to any agent.  The content for the webserver agent will stay as it represents the minimum about of info a user needs to configure the contrast-service in the role of a webserver agent.  It would easier and less confusing new users than directing them to a global/generic doc of contrast-service overall config possibilities.